swap_meet/item.py

```diff
@@ -0,0 +1,29 @@
+class Item:
+
+    def __init__(self, category = None, condition = None):
+        if category == None:
+            self.category = ""
+        else:
+            self.category = category
+
+        if condition == None:
+            self.conditionn = 0
+        else:
+            self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
```

Comment:
👍 will there potentially be decimal values, like 2.5? If not, we could change the if statements to `if self.condition == 0` rather than a range, but this works just fine!